freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -269,24 +307,125 @@ def handle(self, event):
                 to_rebuild_digests.add(digest)
 
         if not to_rebuild_digests:
-            msg = f"No bundle images to rebuild for advisory {event.advisory.name}"
+            msg = f"No bundle images to rebuild for advisory {self.event.advisory.name}"
             self.log_info(msg)
             db_event.transition(EventState.SKIPPED, msg)
```

Comment:
Hi @apaplaus , could you update the event with a consistent string at the beginning of reason string? Release-Driver needs to check this reason string to know whether there is something wrong or just there is no bundle image need to be rebuilt. For example, something like "No bundle images to rebuild for advisory ...: <exact reason>".